tests/test_bibliotheca.py

```diff
@@ -932,19 +977,19 @@ def test_exception(self):
         error = parser.process_all(self.TRIED_TO_HOLD_BOOK_ON_LOAN)
         assert isinstance(error, CannotHold)
 
-
-class TestBibliothecaEventMonitor(BibliothecaAPITest):
+class TestBibliothecaPurchaseMonitor(BibliothecaAPITest):
```

Comment:
Most of the EventMonitor tests end up in the PurchaseMonitor test class because they focused on the ability to set a custom start date, something that now lives in PurchaseMonitor.